src/tests/eth/istanbul_builtins.json

```diff
@@ -0,0 +1,52 @@
+{
+  "0000000000000000000000000000000000000001": { "name": "ecrecover", "pricing": { "linear": { "base": 3000, "word": 0 } } },
+  "0000000000000000000000000000000000000002": { "name": "sha256", "pricing": { "linear": { "base": 60, "word": 12 } } },
+  "0000000000000000000000000000000000000003": { "name": "ripemd160", "pricing": { "linear": { "base": 600, "word": 120 } } },
+  "0000000000000000000000000000000000000004": { "name": "identity", "pricing": { "linear": { "base": 15, "word": 3 } } },
+  "0000000000000000000000000000000000000005": { "name": "modexp", "activate_at": "0x00", "pricing": { "modexp": { "divisor": 20 } } },
+  "0000000000000000000000000000000000000006": {
+    "name": "alt_bn128_add",
+    "pricing": {
+      "0": {
+        "price": { "alt_bn128_const_operations": { "price": 500 }}
+      },
+      "0": {
+        "info": "EIP 1108 transition",
+        "price": { "alt_bn128_const_operations": { "price": 150 }}
+      }
+    }
+  },
+  "0000000000000000000000000000000000000007": {
+    "name": "alt_bn128_mul",
+    "pricing": {
+      "0": {
+        "price": { "alt_bn128_const_operations": { "price": 40000 }}
+      },
+      "0": {
+        "info": "EIP 1108 transition",
+        "price": { "alt_bn128_const_operations": { "price": 6000 }}
+      }
+    }
+  },
+  "0000000000000000000000000000000000000008": {
+    "name": "alt_bn128_pairing",
+    "pricing": {
+      "0": {
+        "price": { "alt_bn128_pairing": { "base": 100000, "pair": 80000 }}
+      },
+      "0": {
+        "info": "EIP 1108 transition",
+        "price": { "alt_bn128_pairing": { "base": 45000, "pair": 34000 }}
+      }
+    }
```

Comment:
I think these should be `"0":` and then `"1:"`. This was taken from EVM tests, right? It is a current issue there that hasn't been resolved yet.

Reply:
See: https://github.com/rust-blockchain/evm-tests/issues/3

Reply:
I don't use that `json file`.
I used aurora precompiles "backend".

Reply:
@jakestutzman also I can't run tests on upstream. I told it before.
So I can't compare. ((

Reply:
Ok then just delete the file.